sardine/src/ffi.rs

```diff
@@ -11,6 +11,11 @@ pub extern "C" fn Srd_New(is_server: bool) -> *mut Srd {
     Box::into_raw(Box::new(Srd::new(is_server))) as *mut Srd
 }
 
+#[no_mangle]
+pub extern "C" fn Srd_NewMultiDelegation() -> *mut Srd {
```

Comment:
We can either break the API and accept a u32 flags parameter, or add a new Srd_NewEx(u32 flags) constructor